README.md

```diff
@@ -61,17 +61,18 @@ For example, this `eth-network-params.json` adds a second node, running a differ
 	"global_client_log_level": "info"
 }
 ```
-To run the package with your desired configuration, simply run
+To run the package with your desired configuration (as specified in your `eth-network-params.json` file, simply run:
```

Comment:
nit: can we we close the parenthesis? `...(as specified in your `eth-network-params.json` file), simply run:`

Reply:
Thanks for catching my silly mistake, will do!
```suggestion
To run the package with your desired configuration (as specified in your `eth-network-params.json` file), simply run:
```